site/zenodo_rdm/moderation/percolator.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2017-2024 CERN.
+# Copyright (C) 2022 Graz University of Technology.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Percolator."""
+
+
+from flask import current_app
+from invenio_search import current_search_client
+from invenio_search.utils import build_alias_name, build_index_name
+
+
+def create_percolator_index(index_name, record_cls):
```

Comment:
```suggestion
def create_percolator_index(record_cls):
```

minor: the percolator index prefix should be in config (e.g. `MODERATION_PERCOLATOR_INDEX_PREFIX = "moderation-queries-"`). We can then reuse the config below to avoid hard-coding.

For the `index_name` param, we can either remove it entirely or make it optional (and default to the config value)